components/bio-formats-tools/test/loci/formats/tools/ImageConverterTest.java

```diff
@@ -124,6 +133,25 @@ public void testDefault(String suffix) throws FormatException, IOException {
     }
   }
 
+  @Test(dataProvider = "options")
+  public void testOptions(String options) throws FormatException, IOException {
+    outFile = File.createTempFile("test", ".ome.tiff");
+    outFile.delete();
```

Comment:
I'm not aware of anything great from Guava: but what's the requirement? For instance, should the File instance correspond to a writable path even if it's not written? Or could it be any old nonsense so long as it's unique?